docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory into the container can lead to potential issues if there are unnecessary files. Ensure your `.dockerignore` file is correctly configured to exclude files that should not be included in the Docker image.